app/config.c

```diff
@@ -129,6 +158,14 @@ static void parse_option(const char *option, const char *params)
         } else if (strncmp(params, "high", 5) == 0) {
             power_save = POWER_SAVE_HIGH;
         }
+    } else if (strncmp(option, "console", 8) == 0) {
+        if (params != NULL) {
+            parse_serial_params(params);
+        }
+    } else if (strncmp(option, "nobench", 8) == 0) {
```

Comment:
Do we want to keep alphabetical ordering ? In which case the previous location for the nopause and nobench flag ordering was better.